<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Email_tasks</class>
 <widget class="QDialog" name="Email_tasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Email Εργασιών</string>
  </property>
  <widget class="QDateEdit" name="dateEditfrom">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d/M/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d/M/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Από Ημερομηνία</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Εως Ημερομηνία</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>81</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSend">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
